lib/node_modules/@stdlib/napi/argv-complex64/include/stdlib/napi/argv_complex64.h

```diff
@@ -0,0 +1,80 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef STDLIB_NAPI_ARGV_COMPLEX64_H
+#define STDLIB_NAPI_ARGV_COMPLEX64_H
+
+#include "stdlib/assert/napi/status_ok.h"
+#include <node_api.h>
+
+/**
+* Macro for converting an add-on callback argument to a single-precision complex floating-point number.
+*
+* @param env      environment under which the function is invoked
+* @param name     output variable name for the array
+* @param argv     name of the variable containing add-on callback arguments
+* @param index    argument index
+*
+* @example
+* #include "stdlib/napi/argv_complex64.h"
```

Comment:
You are missing the same header in this example, as well. Please refer to https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/napi/argv-double/include/stdlib/napi/argv_double.h and investigate accordingly.